docker/docker-compose.pro.dev.yml

```
@@ -0,0 +1,42 @@
+﻿version: "3.7"
+services:
+  backend:
```

```suggestion
  backend-pro:
```
Let's use a different service name, so that both can run in parallel, if the user wants them to.

---

If I replace backend by backend-pro, I'm not able to build it:

version: "3.7"
services:
  backend-pro:
    image: productopener-backend-pro-dev
    build:
      context: ../
      dockerfile: ./docker/backend/Dockerfile
      target: runnable
    volumes:
      - type: bind
        source: ./backend-pro-dev/scripts/
        target: /opt/scripts/

stephane@stephane-hp:~/openfoodfacts-server/docker$ ./build_pro_dev.sh 
backend uses an image, skipping

Maybe I also need to duplicate the "backend" dir in "backend-pro"? Which means all the Apache modules etc. would need to be installed for this new backend-pro as well?